src/common/textcmn.cpp

```diff
@@ -175,6 +176,7 @@ void wxTextAttr::Init()
     m_textEffectFlags = wxTEXT_ATTR_EFFECT_NONE;
     m_outlineLevel = 0;
     m_bulletNumber = 0;
+    m_fontUnderlined = false;
```

Comment:
`m_fontUnderlined` is initialized twice. `m_colUnderline` is not initialized.

Reply:
@MaartenBent 
Double initialization removed.
Can you think of the default value this member can be initialized for all 3 platforms?

Reply:
I would say `wxNullColour` (and also as the default argument in `SetFontUnderline`). This way, the text colour can be used if the underline colour is not explicitly set.
I have only looked at the msw implementation, I don't know how gtk and osx differentiate between using the text colour for the underline or using a user specified colour.

Reply:
OK, fixed as suggested.
I tried Windows version and it seems to work.

I will try *nix/OSX as well later.